### PR TITLE
Added React Roter, Page folder

### PR DESCRIPTION
**Pull request by paulavanina**

### CODE REVIEW

File: src/components/Features/FeaturesImages.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  {
+    image: '',
+    title: 'Titlu',
+    description: 'Descriere',
+  },
+];
+
+export function FeaturesImages() {
+  const items = data.map((item) => (
+    <div className={classes.item} key={item.image}>
```
Comment (milosanca) [inline src/components/Features/FeaturesImages.tsx]: add an ID to the data array and ise the ID as key instead of the image

**Review by milosanca — CHANGES_REQUESTED** [review]

File: src/components/HeroHeader/HeroHeader.module.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.inner {
+  display: flex;
+  justify-content: space-between;
+  /* padding-top: calc(var(--mantine-spacing-xl) * 4); */
```
Comment (silviuvld) [inline src/components/HeroHeader/HeroHeader.module.css]: why we have comments in code?

File: src/App.tsx [hunk 3]
```diff
@@ -1,34 +1,29 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+// import { HeaderMegaMenu } from "./components/Header/HeadeMegaMenu"
```
Comment (silviuvld) [inline src/App.tsx]: why we have comments in code?

File: src/App.tsx [hunk 4]
```diff
@@ -1,34 +1,29 @@
...
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+   <div>
+    {/* <HeaderMegaMenu />
```
Comment (silviuvld) [inline src/App.tsx]: why we have comments in code?

**Review by silviuvld — CHANGES_REQUESTED** [review]

**Review by milosanca — APPROVED** [review]

**Pull request merged**